docs/source/users/options/minimizer_option.rst

```diff
@@ -583,7 +583,8 @@ uncertainties.
 We interface via the python interface `iminuit <https://iminuit.readthedocs.io>`__ with
 support for the 2.x series.
 
-* `Minuit's MIGRAD <https://root.cern.ch/root/htmldoc/guides/minuit2/Minuit2.pdf>`__ (:code:`minuit`)
+* `Minuit's MIGRAD <https://root.cern.ch/root/htmldoc/guides/minuit2/Minuit2.pdf>`__ (:code:`migrad`)
+* `Minuit's SIMPLEX <https://root.cern.ch/root/htmldoc/guides/minuit2/Minuit2.pdf>`__ (:code:`simplex`)
```

Comment:
Both these links point to the same place - and it's not immediately clear when opening the guide where the relevant information about each minimizer is. It may be worth instead pointing to the iminuit docs? https://iminuit.readthedocs.io/en/stable/reference.html#iminuit.Minuit.migrad
https://iminuit.readthedocs.io/en/stable/reference.html#iminuit.Minuit.simplex